Amplitude.js

```diff
@@ -86,7 +90,10 @@
 
         function setUserIdentity(id, type) {
             if (isInitialized) {
-                if (type === window.mParticle.IdentityType.CustomerId) {
+                if (forwarderSettings.userIdentification === constants.MPID) {
```

Comment:
small thing here to discuss really quick

since i am trying to keep this both pre and post idsync-compatible, i made a call to say if client is on v1 and choose MPID, then we set all identities as attributes...upon just typing that out, i feel like maybe it should remain how it was before. anyhow, before i revert let's chat.